coreapi/main_api.py

```diff
@@ -120,7 +120,7 @@ def nsfwclassifier(request, filename):
 
     logger.info(msg="nsfwclassifier called")
     file_path = os.path.join(MEDIA_ROOT, 'images/' + filename)
```

Comment:
os.path.join(MEDIA_ROOT, 'images' , filename)

Reply:
done. similarly for other places as well